nemo/export/vllm/model_config.py

```diff
@@ -223,3 +228,9 @@ def _load_hf_arguments(self, nemo_config: Dict[str, Any]) -> Dict[str, Any]:
                     break
 
         return hf_args
+
+    def try_get_generation_config(self, *args, **kwargs):
```

Comment:
We recently added saving `generation_config.json` in `context/artifacts` subdir, see https://github.com/NVIDIA/NeMo/pull/12403. Not sure if this is meant here, but we could try loading it.